hicommonwealth/commonwealth
# Aave Proposal Loading from CW API

## Link to Issue
Closes: #4697

## Description of Changes
- Added 2 new routes in CW API:
    - /proposals
    - /proposalVotes
- Initialize proposals using data returned from new routes instead of `/entities`
- Refactored the `Aave` API controller so that loading the executors to load proposals is no longer necessary.
- Use ReactQuery to load proposals and votes

## Test Plan
- Visit the `dydx` proposal page.
- Load any proposal and view votes. Refresh the page to ensure a specific proposal page loads directly.
- Compare the proposal page and proposal votes to production
- Load the osmosis proposals page and ensure it is unaffected.

## Deployment Plan

1. 

## Other Considerations

-
